facefusion/utilities.py

```diff
@@ -75,18 +75,37 @@ def create_video(target_path : str, fps : float = 30) -> bool:
 	return run_ffmpeg(commands)
 
 
-def restore_audio(target_path : str, output_path : str) -> None:
+def restore_audio(target_path : str, output_path : str, fps: int) -> None:
 	trim_frame_start = facefusion.globals.trim_frame_start
 	trim_frame_end = facefusion.globals.trim_frame_end
 	temp_output_path = get_temp_output_path(target_path)
-	commands = [ '-hwaccel', 'auto', '-i', temp_output_path, '-i', target_path ]
-	if trim_frame_start is not None and trim_frame_end is not None:
-		commands.extend([ '-filter:v', 'select=between(n,' + str(trim_frame_start) + ',' + str(trim_frame_end) + ')' ])
-	elif trim_frame_start is not None:
-		commands.extend([ '-filter:v', 'select=gt(n,' + str(trim_frame_start) + ')' ])
-	elif trim_frame_end is not None:
-		commands.extend([ '-filter:v', 'select=lt(n,' + str(trim_frame_end) + ')' ])
-	commands.extend([ '-c:a', 'copy', '-map', '0:v:0', '-map', '1:a:0', '-y', output_path ])
+
+	# Create temp audio file extracted from target video
+	temp_target_audio_path = os.path.join(os.path.dirname(target_path), Path(target_path).stem + '.mp3')
+	commands = ['-hwaccel', 'auto', '-i', target_path, '-vn', '-y', temp_target_audio_path ]
+	done = run_ffmpeg(commands)
+	if not done:
```

Comment:
I'm against this coding style - no termination within a method as it then lies about what it's doing

use this instead:

done = run_ffmpeg(extract_audio_commands) ann run_ffmpeg(merge_audio_commands) in the end of the file